### ISSUE TITLE
Cannot transfer data from SD-memory card to Advanced AR glasses

### ISSUE THREAD

**Issue by FellsHollow** [CleverRaven/Cataclysm-DDA#68551]
### Describe the bug

Tried to transfer data from a device to a memory card and then to a new device and it did not work as intended.

### Attach save file

[Anston-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/12839591/Anston-trimmed.tar.gz)

### Steps to reproduce

I have an e-Ink tablet PC with 50 books on it. I tried to copy the data onto another SD memory card (clean), which gave me a successful message. 

The SD card I copied _to_ still showed as an SD memory card (clean) after the transfer. 

So, I tried to use my advanced AR glasses to download the data on that memory card despite it saying it was clean and got the message that no new data was found on the SD memory card (clean). 

I then tried to access the books from my advanced AR glasses and it said "You have nothing to read".

### Expected behavior

I expected to be able to transfer my book collection (50 books) from one device to another using an SD memory card and be able to get rid of the old device (a tablet) to only use the new device (Advanced AR glasses).

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.3448 (22H2)
- Game Version: d6ec466 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions]
]

**Comment by FellsHollow** [comment]
Hello. An update: I managed to download from the SD memory card after I went to another location and found another clean SD card. I am not sure how it was fixed but what I did do was copy from the advanced AR glasses onto the new memory card (only 4 books from the glasses) and then tried to copy from the SD card with the 50 books on it, and it worked fine.

I would still suggest looking into the issue if anyone has time because it definitely did not work at the time I posted this thread.
Thanks!
